### PR TITLE
Beekeepers fetch endpoint accepts data on POST request

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

Replace the dummy data the endpoint used with real data passed in on the POST request.
The data format should be compatible with the front end whereby we will pass in the leaflet geom unmodified `{lat: 2, lng: 1}`, the "RASTERS" constant for the filenames, and eventually also the forage range as 3k or 5k (subj to change). We don't have all the Rasters yet from the client, so file failures will return `0` for the data rather than fail for files that can't be found.

Connects #308 

### Demo

After submitting the JSON provided in the testing section
screen shot 2018-11-06 at 11 00 44 am

## Testing Instructions

Submit data with the required params to the fetch endpoint at http://localhost:8000/beekeepers/fetch/
The coordinates should be in Pennsylvania

```
{
    "forage_range": "3k",
    "location": {
        "lat": 40.741,
        "lng": -76.123
    },
    "filenames": {
        "PESTICIDE": "pesticide",
        "FORAGE_SPRING": "forage_spring",
        "FORAGE_SUMMER": "forage_summer",
        "FORAGE_FALL": "forage_fall"
    }
}
```

### CODE REVIEW

**Comment by rajadain** [comment]
Taking a look now.

**Comment by fungjj92** [comment]
Good point! I will have the request and response mirror one another.

**Comment by fungjj92** [comment]
screen shot 2018-11-07 at 1 17 59 pm


With this data input
```
{
    "forage_range": "3k",
    "location": {
        "lat": 40.741,
        "lng": -76.123
    },
    "indicators": ["hive_density", "habitat", "pesticide", "forage_spring", "forage_summer"]
}
```

**Comment by rajadain** [comment]
Taking another look.

File: src/icp/apps/beekeepers/views.py [hunk 1]
```diff
@@ -11,43 +11,28 @@
...
 @decorators.permission_classes((AllowAny, ))
 def fetch_data(request):
     """
-    Return the cell values from rasters on s3.
+    Return the cell values from rasters on s3 and any errors.
 
-    TODO: Receive validated geom, forage range, and filename(s) from the
-    request
+    :param location: Dict with lat and lng values, i.e {lat: float, lng: float}
+    :param forage_range: A string value of "3km" or "5km"
```
Comment (fungjj92) [inline src/icp/apps/beekeepers/views.py]: sorry getting confused between PRs. these should be "3k" or "5k". to be updated in #324 to Xkm

File: src/icp/apps/beekeepers/views.py [hunk 2]
```diff
@@ -11,43 +11,28 @@
...
-    geom = {
-        'lat': 39.9473832311,
-        'lon': -76.9297742315,
-    }
-    forage_range = '3k'
 
-    filenames_at_buffer = FILENAMES[forage_range]
+    location = request.DATA['location']
+    forage_range = request.DATA['forage_range']
+    indicators = request.DATA['indicators']
```
Comment (rajadain) [inline src/icp/apps/beekeepers/views.py]: Consider calling this `rasters` to be consistent with the front-end.

Comment (rajadain) [reply src/icp/apps/beekeepers/views.py]: That works too!

**Pull request merged**